src/Compilers/CSharp/Portable/Symbols/OverriddenOrHiddenMembersHelpers.cs

```diff
@@ -262,8 +262,8 @@ bool isAccessorOverride(MethodSymbol accessor, MethodSymbol overriddenAccessor)
                     return MemberSignatureComparer.CSharpAccessorOverrideComparer.Equals(accessor, overriddenAccessor); //NB: custom comparer
                 }
 
-                if (accessor is PEMethodSymbol { ExplicitlyOverriddenClassMethod: MethodSymbol peOverriddenAccessor } &&
-                    overriddenAccessor.Equals(peOverriddenAccessor, TypeCompareKind.AllIgnoreOptions))
+                if (accessor is MethodSymbol { ExplicitlyOverriddenClassMethod: MethodSymbol explicitlyOverriddenAccessor } &&
```

Comment:
>accessor is MethodSymbol [](start = 20, length = 24)

It looks like ```accessor``` is already a method symbol and the type check is redundant.

Reply:
No additional code is produced. This is written for clarity.

---
In reply to: [424656304](https://github.com/dotnet/roslyn/pull/44025#discussion_r424656304) [](ancestors = 424656304)